docs/application/dotnet/tutorials/hybrid/create-dotnet-hybrid-package.md

```diff
@@ -0,0 +1,111 @@
+![Create Web Widget 4](./media/create_web_sample_cs_4.png)
+
+2. Check manifest file of web application
+
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <widget xmlns="http://www.w3.org/ns/widgets" xmlns:tizen="http://tizen.org/ns/widgets" id="http://yourdomain/WorldClockWidget" version="1.0.0" viewmodes="maximized">
+        <access origin="*" subdomains="true"></access>
+        <tizen:app-widget id="TestHybrid.WorldClockWidget.WorldClockWidget" primary="true">
+            <tizen:widget-label>World Clock Widget</tizen:widget-label>
+            <tizen:widget-content src="widget/Widget/index.html">
+                <tizen:widget-size preview="widget/Widget/preview.png">2x2</tizen:widget-size>
+            </tizen:widget-content>
+        </tizen:app-widget>
+        <tizen:application id="TestHybrid.WorldClockWidget" package="TestHybrid" required_version="2.3.2"/>
+        <content src="index.html"/>
+        <feature name="http://tizen.org/feature/screen.size.all"/>
+        <icon src="icon.png"/>
+        <name>WorldClockWidget</name>
+        <tizen:privilege name="http://tizen.org/privilege/application.launch"/>
+        <tizen:profile name="wearable"/>
+    </widget>
+    ```
+    package id of web application is set like below :
+     __*package="TestHybrid"*__
+
+3. Create .NET Sample Application
+![Create .NET App 1](./media/create_dotnet_sample_cs_1.png)
+![Create .NET App 2](./media/create_dotnet_sample_cs_2.png)
+
+4. Check manifest of .NET application
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <manifest xmlns="http://tizen.org/ns/packages" api-version="4" package="org.tizen.example.CrossTemplate1.Tizen" version="1.0.0">
+    <profile name="common" />
+    <ui-application appid="org.tizen.example.CrossTemplate1.Tizen"
+                    exec="CrossTemplate1.Tizen.dll"
+                    type="dotnet"
+                    multiple="false"
+                    taskmanage="true"
+                    nodisplay="false"
+                    launch_mode="single">
+    <label>CrossTemplate1.Tizen</label>
+    <icon>CrossTemplate1.Tizen.png</icon>
+    </ui-application>
+    </manifest>  
+    ```
+    package id of .NET application is set like below :
+    __*package="org.tizen.example.CrossTemplate1.Tizen"*__
+
+5. Create hybrid package with CLI tool
+    ```bash
+    /home/test/tizen-studio/tools/ide/bin$ ./tizen package -t wgt -s test_cert_profile -r /home/test/hybrid/org.tizen.example.CrossTemplate1.Tizen-1.0.0.tpk – /home/test/hybrid/WebWidgetSample.wgt
+
+    Author certficate: /home/test/tizen-studio-data/keystore/author/test_cert_profile.p12
+    Distributor1 certificate : /home/test/tizen-studio/tools/certificate-generator/certificates/distributor/sdk-partner/tizen-distributor-signer.p12
+    Package( /home/test/hybrid/WebWidgetSample.wgt ) is created successfully.
+    ```
+    *[Note] Newly generated hybrid application package file overrides previous web application package file.*
```

Comment:
Is this Note? If so, please apply note style.

Reply:
done